libs/api/src/common/config.ts

```diff
@@ -0,0 +1,12 @@
+export const siteConfig =
+{
+    CHAIN_NAME: "Optimism Goerli",
+    CHAIN_ID: "0x1a4",
+    NETWORK: "opt-goerli",
+    RPC_URL: "https://goerli.optimism.io",
+    L1_BLOCK_EXPLORER_URL: "https://goerli.etherscan.io",
+    L2_BLOCK_EXPLORER_URL: "https://goerli-optimism.etherscan.io",
+    LOGO_BADGE: "Goerli",
+    BACKEND_URL: "https://hd-backend-api.herokuapp.com",
+    BACKEND_TOKEN: "cc919b6a43b0921fc30fd3a6dbfdd9cdd636ffa9",
```

Comment:
What are your plans for later? We don't wanna deploy with the token in the public repo, I assume

Reply:
> What are your plans for later? We don't wanna deploy with the token in the public repo, I assume

let me change this, also create new token for test environment